src/containers/forms/PractitionerForm/UserIdSelect/tests/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import React from 'react';
+import UserIdSelect from '..';
+import { OpenSRPService } from '../../../../../services/opensrp';
+import { response } from './fixtures';
+
+// tslint:disable-next-line: no-var-requires
+const fetch = require('jest-fetch-mock');
+
+describe('src/*/forms/userIdSelect', () => {
```

Comment:
I feel that these tests are insufficient.  How can we include the fixtures in the tests?  Can we test that the list of users is rendered?

Reply:
will work on the tests in #526